homeassistant/components/gpsd/coordinator.py

```diff
@@ -0,0 +1,64 @@
+"""Data coordinator for the GPSD integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+from typing import Any
+
+from gps3.agps3threaded import AGPS3mechanism
+
+from homeassistant.const import (
+    ATTR_ELEVATION,
+    ATTR_LATITUDE,
+    ATTR_LONGITUDE,
+    ATTR_MODE,
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PORT,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import ATTR_CLIMB, ATTR_GPS_TIME, ATTR_SPEED
+
+UPDATE_INTERVAL = timedelta(minutes=1)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class GpsdCoordinator(DataUpdateCoordinator):
```

Comment:
Please add the return type here as well in DataUpdateCoordinator[...]

Reply:
Hope it's correct now